Circle packing viz for test

dongnizh opened the pull request:
Hi @chrismattmann :

The steps of generating that circle packing viz are (on circle-packing branch right now)

```
1. python similarity.py -f [folder of images]
2. python circle-packing.py 
3. open circlepacking,html

```

Hopefully it works well.

chrismattmann commented:
hey @dongnizh this only works for .jpg is that intended? But you made it so that similarity and other things only work for .jpg? was that intended?

dongnizh commented:
Hi, @chrismattmann I deleted that ".jpg" suffix checking by mistake at the first commit. So that I added back at the second commit. In the future, it would be working for any type of images. Some more need to talk to you when I get back. This is just for testing.

chrismattmann commented:
got it, OK merged!
